templates/index.html

```diff
@@ -17,25 +17,38 @@ <h2>Items</h2>
                             <div class="col-8">{{ item.title }}</div>
                             <div class="col-2">
                                 {% if item.status == 'Not Started' %}
-                                    <form id="complete_item" method="post" action="">
-                                        <a href="#">
-                                    <span class="badge badge-danger badge-pill red"
-                                          onclick="complete_item({{ item.id }})">{{ item.status }}</span>
-                                        </a>
-                                    </form>
+                                    <button class="badge badge-danger badge-pill red shadow-none"
+                                            style="border: none">
+                                        {{ item.status }}
+                                    </button>
```

Comment:
Why is this a `<button>`? It doesn't do anything on click.

Reply:
Forgot to remove it after extracting the button